sql/plan/filter.go

```diff
@@ -55,10 +55,16 @@ type filterIter struct {
 func (i *filterIter) Next() (sql.Row, error) {
 	for {
 		row, err := i.childIter.Next()
+		if err != nil {
+			return nil, err
+		}
 
+		ok, err := i.f.expression.Eval(row)
 		if err != nil {
 			return nil, err
-		} else if i.f.expression.Eval(row) == true {
+		}
+
+		if ok == true {
```

Comment:
with `if ok {` I think the result is the same :)

Reply:
Nope: https://play.golang.org/p/XP_hLd18CJF

Reply:
Nope: https://play.golang.org/p/XP_hLd18CJF

Reply:
So, at least for me, have a `ok` variable name that usually means boolean feels a little bit strange for me. Maybe we should change the variable name for somethig less common, like `evaluated` ?

Reply:
Yes, that makes sense. `ok` might induce to think it's a bool